src/hooks/use-window-listener.ts

```diff
@@ -1,6 +1,9 @@
 import { useEffect } from 'react';
 
-export function useWindowListener( type: keyof WindowEventMap, callback: () => void ) {
+export function useWindowListener(
+	type: keyof WindowEventMap,
+	callback: Parameters< typeof window.addEventListener >[ 1 ]
```

Comment:
This is a leftover from when I tried implementing this dialog using the `beforeunload` window event (which didn't work, because Chromium/Electron blocked the confirmation modal from displaying). I still left the change in because this is a more correct type definition.